packages/sdk/src/client/session/decorators/wallet.ts

```diff
@@ -33,11 +34,19 @@ export function zksyncSsoWalletActions<
         delete unformattedTx.eip712Meta;
       }
 
+      /* eslint-disable @typescript-eslint/no-unused-vars */
+      const { chainId: _, ...unformattedTxWithPaymaster } = await getTransactionWithPaymasterData(
```

Comment:
I think we don't need that here. We just need to check if transaction already has paymaster. If not, then get client.paymasterHandler() and update paymasterInput and paymasterAddress fields on the transaction

Reply:
yes, that's what `getTransactionWithPaymasterData` does. It check if transaction already paymaster and if not, calls paymasterHandler. I'd suggest keeping `getTransactionWithPaymasterData` function since its used in many places